packages/web/docs/src/components/lede.tsx

```diff
@@ -0,0 +1,7 @@
+import { cn } from '@theguild/components';
+
+export interface LedeProps extends React.HTMLAttributes<HTMLParagraphElement> {}
+
+export function Lede(props: LedeProps) {
```

Comment:
Word of the day?

<img width="622" alt="image" src="https://github.com/user-attachments/assets/f527a65d-63e7-4dc1-8439-42819ed9c85f" />